Main.cpp

```diff
@@ -134,9 +134,12 @@ class VoiceCodec_OPUS_PLC {
 			}
 			//LOG_MESSAGE(PLID, "Normal frame");
 
-			int16_t decodedSamples;
-			decodedSamples = opus_decode(_opusDecoder, &encodedBytes[curEncodedBytePos], payloadSize, &rawSamples[decodedRawSamples], maxRawSamples - decodedRawSamples, 0);
+			int16_t decodedSamples = opus_decode(_opusDecoder, &encodedBytes[curEncodedBytePos], payloadSize, &rawSamples[decodedRawSamples], maxRawSamples - decodedRawSamples, 0);
 
+			if (decodedSamples <= 0) {

```

Comment:
As I can see `0` is ok here. So it should be:
```suggestion
			if (decodedSamples < 0) {
```

Reply:
r sure?

Reply:
Yes.

Reply:
Btw im not, coz valve also check for <=, coz 0 - impossible, non valid and nothing? wtf?

Reply:
Zero - is zero, nothing. You  can try to emulate it.

Reply:
ladno, poka.